packages/add/index.js

```diff
@@ -11,6 +11,13 @@ const modifyHelper = require("@webpack-cli/utils/modify-config-helper");
  *
  */
 
-module.exports = function add() {
-	return modifyHelper("add", defaultGenerator);
+module.exports = function add(...args) {
+	const DEFAULT_WEBPACK_CONFIG_FILENAME = "webpack.config.js";
+
+	const filePaths = args.slice(3);
```

Comment:
Why are you slicing 3 here?

Reply:
Because first 3 args would be node path, webpack-cli path, add command.
4th item in the array would be everything after add command like the custom config path/name. So, we're slicing first three items to get the config path.

We can't directly use the last element of `args` because that would be "add" if the user doesn't mention any arg after add.

Reply:
Oki. let me check it out later to make sure its working 👍